extra/prometheus/prometheus.yml

```diff
@@ -1,28 +1,15 @@
 global:
   scrape_interval:     30s # Set the scrape interval to every 30 seconds. Default is every 1 minute.
-  scrape_timeout:      30s # Set the scrape output to every 30 seconds. Default is every 1 minute.
+  scrape_timeout:      30s # Set the scrape timeout to shorter or match or match scrape_interval. Default is every 1 minute.
```

Comment:
now this doesn't make sense. you have match twice

Reply:
Copy, paste and post edit typo.
Fixed in commit [a437f30](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/pull/87/commits/a437f30e6a57c70102a98c6e37caacc903612a57)